app/controllers/articles_controller.rb

```diff
@@ -5,11 +5,11 @@ class ArticlesController < ApplicationController
   before_action :require_same_user, only: %i[edit update destroy]
 
   def index
+    #reset_session
```

Comment:
Are you sure you reset the session here?

Reply:
session' s reset is for testing, or when user table' s data is deleted, session's error came out with "can't find session user_id with 5 or 6, 7,8...."